python/lsst/obs/base/_instrument.py

```diff
@@ -111,6 +118,29 @@ def getName(cls):
         """
         raise NotImplementedError()
 
+    @classmethod
+    def getCuratedCalibrationNames(cls) -> Set[str]:
+        """Return the names of all the curated calibration dataset types.
+
+        Returns
+        -------
+        names : `tuple` of `str`
+            The dataset type names of all curated calibrations. This will
+            include the standard curated calibrations even if the particular
+            instrument does not support them.
+
+        Notes
+        -----
+        The returned list does not indicate whether a particular dataset
+        is present in the Butler repository, simply that these are the
+        dataset types that are handled by ``writeCuratedCalibrations``.
+        """
+
+        # Do not check the data packages for calibrations for now.
+        # Camera is a special dataset type that is also handled as a
+        # curated calibration.
+        return set(("camera",) + cls.standardCuratedDatasetTypes + cls.additionalCuratedDatasetTypes)
```

Comment:
Why does `camera` not belong in `standardCuratedDatasetTypes`? Special-casing something like this seems like it could cause bugs later.

If the point is that these are the curated calibrations only for conversion purposes, but not for others, then the method/property name should reflect that.

Reply:
because "standard" means "calibration data found in obs_x_data package following the standard organization". At some point versioned camera geometry will turn up in that form and when it does we will include it in the standard list. If you look at writeCuratedCalbrations you will see it does standard curated, additional curated, and camera as distinct operations.

Reply:
As for the name, I am wanting the list of names returned here to match the list of dataset types that writeCuratedCalibrations will be writing.